app/waf/test/unit/service.OnboardingManager.ts

```diff
@@ -93,7 +93,8 @@ describe('test OnboardingManager', async () => {
   });
 
   it('onboard: ok', async () => {
-    let adc = <Adc>createAdcObject();
+    let adc = new Adc(createAdcObject());
+    doMgr.config.endpoint = adc.getDoEndpoint();
```

Comment:
This should be passed in when doMgr is instantiated.

Reply:
done